Library/Homebrew/dev-cmd/man.rb

```diff
@@ -18,6 +22,11 @@ def man
     else
       regenerate_man_pages
     end
+
+    args = ["-C", HOMEBREW_REPOSITORY, "diff", "--quiet"]
```

Comment:
I'd just inline these as you're not using the var in more than one place and it makes the command easier to follow.